src/Tickets/Commerce/Settings.php

```diff
@@ -292,6 +319,31 @@ public function get_settings() {
 				'validation_type' => 'options',
 				'options'         => $tc_currency_options,
 			],
+
+			static::$option_currency_decimal_separator      => [
+				'type'            => 'text',
+				'label'           => esc_html__( 'Decimal Separator', 'event-tickets' ),
+				'tooltip'         => esc_html__( 'This sets the decimal separator of displayed prices.', 'event-tickets' ),
+				'default'         => '.',
```

Comment:
Use the default using the static variables from `Tickets/Commerce/Utils/Currency`